clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -131,17 +131,10 @@
       },
       {
         "id": "hpcobol",
-        "extensions": [
-          ".hpcbl",
-          ".hpcobol",
-          ".hp",
-          "ecobol"
-        ],
```

Comment:
Won't removing these force VSCode to guess the languageid and accidentally switch to regular cobol (followed by a flood of diagnostics)?

Reply:
Fair point.
 @VitGottwald Do you think its ok? as it was our wild guess of what file extension would belong to hp-cobol

Reply:
That is actually intended. This new language id is added as an experiment and in the typical setting the files do not have a file extension. The users have to explicitly use 
```
	"files.associations": {
		"**/hp-tandem/**": "hpcobol"
	}
```
I asked about file extensions, but was told they would not use them.